nbstripout.py

```diff
@@ -143,15 +143,43 @@ def _cells(nb):
             yield cell
 
 
-def strip_output(nb, keep_output, keep_count):
+def pop_recursive(d, key, default=None):
```

Comment:
A test for this function would be great.

Reply:
added via doctest